Doc/faq/programming.rst

```diff
@@ -16,6 +16,11 @@ Is there a source code level debugger with breakpoints, single-stepping, etc.?
 
 Yes.
 
+Several debuggers for Python are described below, and the built-in function
+:func:`breakpoint` allows you to drop into any of them just by setting the
+:func:`sys.breakpointhook` hook function, conveniently letting you change a
```

Comment:
Hm I worry this may be read the wrong way.  People don’t need to worry about `sys.breakpointhook`, in my understanding it (and the equivalent envvar) are hooks for IDEs / alternate interpreters.  Of course devs can use it too in usercustomize for example to define their preferred debugger, but that’s more advanced.

What do you think?  Would it be enough to mention only breakpoint in this FAQ and let readers follow the link to see more details?

Reply:
Agreed with you on keeping this simple with respect to FAQ. breakpoint docs have same information about customisation where user is taken to https://docs.python.org/3/library/functions.html#breakpoint

Reply:
I agree with you, Éric. I'll update the PR in a moment.

Reply:
Just did the change. Éric, let me know if it is better now :)